src/libs/ReportUtils.js

```diff
@@ -514,6 +524,23 @@ function isExpensifyOnlyParticipantInReport(report) {
     return reportParticipants.length === 1 && _.some(reportParticipants, (accountID) => _.contains(CONST.EXPENSIFY_ACCOUNT_IDS, accountID));
 }
 
+/**
+ * Returns whether a given report can have tasks created in it.
+ * We only prevent the task option if it's a DM/group-DM and the other users are all special Expensify accounts
+ *
+ * @param {Object} report
+ * @returns {Boolean}
+ */
+function canCreateTaskInReport(report) {
+    const otherReportParticipants = _.without(lodashGet(report, 'participantAccountIDs', []), currentUserAccountID);
+    const areExpensifyAccountsOnlyOtherParticipants = _.every(otherReportParticipants, (accountID) => _.contains(CONST.EXPENSIFY_ACCOUNT_IDS, accountID));
```

Comment:
Just a heads up, i believe the logic here caused this [deploy blocker](https://github.com/Expensify/App/issues/30804).

Reply:
Thanks for pointing that out! That's a good catch.